microprofile-fault-tolerance/src/test/java/org/jboss/eap/qe/microprofile/fault/tolerance/MultipleJarDeploymentTest.java

```diff
@@ -0,0 +1,99 @@
+package org.jboss.eap.qe.microprofile.fault.tolerance;
+
+import java.util.regex.Pattern;
+
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.faulttolerance.FallbackHandler;
+import org.hamcrest.CoreMatchers;
+import org.hamcrest.MatcherAssert;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.OperateOnDeployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.as.arquillian.api.ServerSetup;
+import org.jboss.eap.qe.microprofile.fault.tolerance.deployments.v10.HelloFallback;
+import org.jboss.eap.qe.microprofile.fault.tolerance.deployments.v10.HelloService;
+import org.jboss.eap.qe.microprofile.fault.tolerance.deployments.v10.MyContext;
+import org.jboss.eap.qe.microprofile.fault.tolerance.util.FaultToleranceServerSetup;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientProvider;
+import org.jboss.eap.qe.microprofile.tooling.server.log.LogChecker;
+import org.jboss.eap.qe.microprofile.tooling.server.log.ModelNodeLogChecker;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+
+/**
+ * Tests multiple deploy/undeploy of MP FT deployments (just JAR archives). This in in-container test which injects
+ * MP FT service and invokes operations on it.
+ */
+@RunWith(Arquillian.class)
+@ServerSetup(FaultToleranceServerSetup.class) // Enables/Disables fault tolerance extension/subsystem for Arquillian in-container tests
+public class MultipleJarDeploymentTest {
+
+    @Inject
+    HelloService helloService;
```

Comment:
Does this supposed to be package private? If not, please add a `private` access modifier.

Reply:
right, fixed.